x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -648,7 +647,7 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
             />
 
             {/* Only display callout if includeFeatureState is enabled and the snapshot was created by ES 7.12+ */}
-            {semverGt(version, '7.12.0') && isFeatureStatesToggleEnabled && (
+            {versionId > 7120099 && isFeatureStatesToggleEnabled && (
```

Comment:
> IndexVersion does not have any concept of major/minor/patch numbers, it will just be an incrementing number - but for older versions, the indexversion id will be the same as the corresponding release version id.

Are we confident that the new index version values will be >7120099, i.e., I guess the numbers will start incrementing after something like 8100000 (looks like change will be released in 8.10)?

Reply:
The `versionId` field was actually existing before 8.10. Before the change, we had two fields - `versionId` and `version`, where `versionId` was a number like 7120099 and `version` was a string representation of the version like '7.12.0'. After the change, the two fields still exist but now the `version` field is the same as `versionId` except that it is a string.

In this PR, I decided to replace the use of `version` with `versionId` when we check the current version since this would work for all versions (both before and after 8.10) and this might be more future-proof since `versionId` is an integer that is incrementing with every release, while `version` is a string representation of the version and it is more likely to be changed.

I found that the 7.12 `versionId` value is 7120099 by creating a 7.12 deployment and checking the `versionId` field from the Get snapshot response.

Reply:
👍 thanks, Elena!